src/hooks/useBottomSheetRef.js

```diff
@@ -0,0 +1,17 @@
+import { useRef } from 'react';
+
+const useBottomSheetRef = () => {
+  const ref = useRef();
+
+  const handleOpen = () => {
+    ref.current.open();
+  };
+
+  const handleClose = () => {
```

Comment:
Al hacer un hook reutilizable es una buena práctica asegurar equidad referencial y utilizar `useCallback` para los callbacks. De todas maneras, como es nuestro proyecto, si no pensamos utilizar `memo`, esta optimización no tiene sentido.

Reply:
Ahí va, en este caso puede estar bien, no estamos usando `memo` xq para que "valga la pena" tenés que poner `useMemo` y `useCallback` por todos lados también y memorizar un montón de cosas que no me parece bien. Toda optimización trae un costo y no siempre un beneficio que lo justifique. https://kentcdodds.com/blog/usememo-and-usecallback